src/Akka.Persistence.Sql.Linq2Db/Extension.cs

```diff
@@ -0,0 +1,60 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="Extension.cs" company="Akka.NET Project">
+//      Copyright (C) 2013-2022 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.Persistence.Sql.Linq2Db.Journal;
+using Akka.Persistence.Sql.Linq2Db.Snapshot;
+
+namespace Akka.Persistence.Sql.Linq2Db
+{
+    public sealed class Linq2DbPersistence : IExtension
```

Comment:
What's the rationale for creating an `ActorSystemExternsion` here?

Reply:
The biggest advantage is preloading the HOCON configuration. Parsing HOCON is a very expensive operation, for a given HOCON file, parsing should be done once and only once at the start of the plugin lifetime.

Reply:
Second is a centralized place to access the default configurations and this also follows the standardized flow we have on other persistence plugins